### ISSUE TITLE
footer navigation like „Daten“ (Data) are leading to an 404 page

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#931]
footer navigation like „Daten“ (Data) are leading to an 404 page
https://de.openfoodfacts.org

**Comment by hangy** [comment]
https://de.openfoodfacts.org/Gesetzliche%20Hinweise
https://de.openfoodfacts.org/Nutzungsbedingungen
https://de.openfoodfacts.org/Die%20Daten

**Comment by teolemon** [comment]
https://crowdin.com/translate/openfoodfacts/103/en-de#36273

**Comment by TaciteOFF** [comment]
Same problem for the French version because /data/ has been translated to /données/

**Comment by TaciteOFF** [comment]
Same problem for the French version because /data/ has been translated to /données/

**Comment by teolemon** [comment]
Same issue for fr @stephanegigandet 
```
Avez-vous constaté que le lien "Données" de la version française du site OpenFoodFacts ne fonctionne pas ? (La version anglaise marche en revanche)
```

**Comment by lan33** [comment]
I fixed tests for Travis in the PR #1424.
The last check problem that breaks Travis is the linebreak in the file `/html/js/display-tag.js`, eslint asks for **CRLF** instead of **LF**.
